packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -303,11 +306,7 @@ const ComboBox = React.forwardRef((props, ref) => {
         });
 
         const handleFocus = (evt) => {
-          if (evt.target.type === 'button') {
-            setIsFocused(false);
-          } else {
-            setIsFocused(evt.type === 'focus' ? true : false);
-          }
+          setIsFocused(evt.type === 'focus' ? true : false);
```

Comment:
is the ternary necessary here? 🤔
```suggestion
          setIsFocused(evt.type === 'focus');
```